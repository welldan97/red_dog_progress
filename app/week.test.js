import React from 'react';
import renderer from 'react-test-renderer';

import Week from './week';

it('It renders', () => {
  const component = renderer.create(<Week />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
