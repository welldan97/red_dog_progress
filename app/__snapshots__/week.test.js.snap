// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders 1`] = `
<table
  className="table table-bordered"
>
  <thead>
    <tr>
      <th />
      <th>
        Monday
      </th>
      <th>
        Tuesday
      </th>
      <th
        className="table-info"
      >
        Wednesday
      </th>
      <th>
        Thursday
      </th>
      <th>
        Saturday
      </th>
      <th>
        Sunday
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th
        scope="row"
      >
         Meditation 
      </th>
      <td>
        🌀
      </td>
      <td>
        🌀🌀🌀
      </td>
      <td
        className="table-info"
      >
        🌀
      </td>
      <td />
      <td />
      <td />
    </tr>
    <tr>
      <th
        scope="row"
      >
         Jogging 
      </th>
      <td>
        🏃
      </td>
      <td>
        ❌
      </td>
      <td
        className="table-info"
      >
        🏃
      </td>
      <td />
      <td />
      <td />
    </tr>
    <tr>
      <th
        scope="row"
      >
         Work 
      </th>
      <td>
        12 hours
      </td>
      <td>
        5 hours
      </td>
      <td
        className="table-info"
      />
      <td />
      <td />
      <td />
    </tr>
  </tbody>
</table>
`;
